proposal_stages.md

```diff
@@ -0,0 +1,28 @@
+# ROS 2 Production Proposal Stages
+The following stages are also intended to be used as github tags for tracking.
+## Stage 1: Draft
+
+The draft stage represents the initial proposal of a project or task which has been formalised into a github issue for presentation to the community. To move to the next stage, this draft should be presented to the ROS2 Production WG and discussed in relevant channels (e.g slack, github, discourse, etc.), having addressed feedback about the scope, methodology, and tasks required to implement this proposal. 
+
+## Stage 2: Ready
+
+In this stage, the draft has received feedback from the community. It is considered to be in “ready for work” state. Tasks have been clearly defined and broken down into relevant sub-components and subtasks, sufficient such that a principal contributor may begin execution with clear direction. To move to the next stage, there should be a discussion/vote on the proposal’s priority, who will be the principal contributor, and the intended timeline for completion. 
```

Comment:
Feedbacks from March 6th WG meeting:
 in this stage the idea has received initial feedbacks and has been discussed by the WG.
 The proposer should open a PR to add a design document for the idea.
 The design document should be reviewed and approved by the WG participants 
 (TBD define roles, number of approvals, etc).
 
 Eventually it will be merged into the repository.
 This won't immediately indicate that someone is working on this item, but rather that the idea has been reviewed by us and is ready to be worked on.

Reply:
I generally agree that a PR to open a design document would be a good way to formalize the proposal and incorpoarate it in the WG documentation. Perhaps we could then create a separate issue type for tracking of merged (e.g. ready) proposals?

In the WG governance document (taken from the template), there are roles defined for reviewers and approvers. How do we feel about nominating folks (to include self nominations) to utilize these roles and facilitate the process for proposals?